src/date-picker/date-picker-dialog.jsx

```diff
@@ -41,18 +42,6 @@ const DatePickerDialog = React.createClass({
     muiTheme: React.PropTypes.object,
   },
 
-  getDefaultProps: function() {
```

Comment:
Well, the thing is, this component is exported in the `index.js` I have seen multiple issues where people where using this component :grin:.